app/views/agent/CapturePreferenceView.scala.html

```diff
@@ -25,38 +25,76 @@
       errorSummary: ErrorSummary,
       input: Input)
 
+
```

Comment:
Overall the view looks good & it errors correctly when nothing is selected however if you click yes & cause an error on the page it still has the little grey bar beside the red one, and the error message itself should be in bold as well. I'll slack you some screenshots for reference